cpp/src/arrow/flight/client.h

```diff
@@ -92,10 +92,8 @@ class ARROW_FLIGHT_EXPORT FlightWriteSizeStatusDetail : public arrow::StatusDeta
   int64_t actual_;
 };
 
-class ARROW_FLIGHT_EXPORT FlightClientOptions {
+struct ARROW_FLIGHT_EXPORT FlightClientOptions {
  public:
```

Comment:
trivia: can remove `public` if it's a `struct`

Reply:
Yup, thanks for catching that.